_posts/2023-04-19-#27039.md

```diff
@@ -211,7 +211,7 @@ commit: 93c70287a
 17:44 <sipa> I'm not sure if progress is saved during the first block index rebuilding phase, but if past-me says so, it's probably true.
 17:44 <pinheadmz> past-you is so smart
 17:45 <pinheadmz> and is reindex-chainstate then just skipping right to that ssecond step ?
-17:45 <sipa> Exactly. -reindex-chainstate is exactly equivalent to rm -rf ~/.bitcoin/chainstate
+17:45 <sipa> Exactly. -reindex-chainstate is exactly equivalent to rm -rf ~/.bitcoin/chainstate
```

Comment:
```suggestion
17:45 <sipa> Exactly. `-reindex-chainstate` is exactly equivalent to `rm -rf ~/.bitcoin/chainstate`
```

Reply:
This is how irccloud renders it:
<img width="671" alt="image" src="https://user-images.githubusercontent.com/69010457/234129094-94526a37-616a-4c5e-8b74-d7da3c405b94.png">